edited provider-quantinuum with link to latest technical specs (#549)
MicrosoftDocs/quantum-docs: articles/provider-quantinuum.md

## System Model H1

> The System Model H1 generation of quantum computers, Powered by Honeywell, includes two machine targets: H1-1 and H1-2. Both quantum computers have fundamentally the same design and both meet a nominal set of technical requirements. However, they may have system-to-system variability in exact performance and features, such as the maximum number of available qubits. Users are encouraged to test compatibility of their code by submitting jobs to the [Syntax Checker](#syntax-checker) and [System Model H1 Emulator](#system-model-h1-emulator) prior to submitting them to the target machines.  

Reviewer: ```suggestion
The System Model H1 generation of quantum computers, Powered by Honeywell, includes two machine targets: H1-1 and H1-2. Both quantum computers have fundamentally the same design and both meet a nominal set of technical requirements. However, they may have system-to-system variability in exact performance and features, such as the maximum number of available qubits. Users are encouraged to test compatibility of their code by submitting jobs to the [syntax checker](#syntax-checkers) and [System Model H1 Emulator](#system-model-h1-emulators) prior to submitting them to the target machines.  
```

  - H1 machine family: `quantinuum.hqs-lt`
- Target Execution Profile: [Basic Measurement Feedback](xref:microsoft.quantum.target-profiles)

### Technical Specifications

> Technical details for System Model H1 and System Model H1 Emulators are found in Quantinuum's Product Data Sheets, which can be found here: [Quantinuum System Model H1](https://www.quantinuum.com/products/h1).

Reviewer: ```suggestion
Technical details for the System Model H1 and System Model H1 Emulators can be found in Quantinuum's [product data sheets](https://www.quantinuum.com/products/h1).
```
